### ISSUE TITLE
mapbox-gl Version 2

### ISSUE THREAD

**Issue by romanstingler** [mapbox/mapbox-gl-draw#1030]
please add support for mapbox-gl 2

```
warning " > @mapbox/mapbox-gl-draw@1.2.0" has incorrect peer dependency "mapbox-gl@>=0.27.0 <2.0.0".
warning "@mapbox/mapbox-gl-draw > eslint-plugin-import@2.22.1" has unmet peer dependency "eslint@^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0".
warning " > @mapbox/mapbox-gl-geocoder@4.7.0" has incorrect peer dependency "mapbox-gl@>= 0.47.0 < 2.0.0".
```

**Comment by awulkan** [comment]
I find it quite worrisome that Mapbox cares so little about this repository (especially now that Mapbox is even more expensive!). This repository is in big need of maintenance, but still only receives a handful of commits each year. I don't know what to do, migrate our company away from Mapbox? Fork this repo and maintain it myself? Constantly ping the devs to merge stuff and keep their code up-to-date?

I don't want to sound like an ass, but I honestly don't know what to do.

**Comment by jaumesala** [comment]
I’m in the same situation. This package is very useful in conjunction with mapbox-gl. I’m a big fan of Mapbox and I’m happy to pay for their products/tools, but I really don’t understand why they are taking care of this repos.

**Comment by bradgreens** [comment]
My company pays for Mapbox Support and I sent them a support ticket about this `->`

> It appears the recent Mapbox GL 2.0 release was not coordinated with useful plugins our Company uses, like Mapbox GL Draw. 
> 
> Does the Mapbox development team intend to assist in this upgrade? Or, is the community on their own to make their plugin compatible with the changes to the 2.0 library?
> 
> Conversation is here on GitHub:
> https://github.com/mapbox/mapbox-gl-draw/issues/1030

**Comment by arindam1993** [comment]
👋 This should now be fixed with `1.2.1`
